integration/travis/wait-for-cook.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# Usage: wait-for-cook.sh [PORT]
+#
+# Examples:
+#   timeout 60s wait-for-cook.sh 12321
+#   timeout 60s wait-for-cook.sh
+#
+# Waits for cook to be listening on the provided port (defaults to 12321).
```

Comment:
Making it a function sounds good. As far as I can tell, `curl` doesn't support retry on connection refused.